README.md

```diff
@@ -25,3 +27,39 @@ JdbcStorageFactory.newBuilder()
                   .setDataSource(dataSource)
                   .build();
 ```
+
+### SQL Type Mapping
+
+Data types differ for various SQL databases. There is `TypeMapping` to deal with it.
```

Comment:
The framework provides a `TypeMapping` to configure the SQL types, which fit the target storage.